tests/datafile/NGC6440E.itoa

```diff
@@ -0,0 +1,62 @@
+1748-202153709.8096123252915 13.88  1949.6090  0.000000  GB          
+1748-202153740.5674746683603 26.89  1949.6090  0.000000  GB          
+1748-202153740.7745986861214 18.29  1949.6090  0.000000  GB          
+1748-202153801.3860512006244 21.48  1949.6090  0.000000  GB          
+1748-202153801.5914330076661 17.88  1949.6090  0.000000  GB          
+1748-202153833.2978103024970 18.59  1949.6090  0.000000  GB          
+1748-202153833.5024577166286 19.03  1949.6090  0.000000  GB          
+1748-202153843.3320793752525 15.07  1949.6090  0.000000  GB          
+1748-202153865.1847677801510 21.58  1949.6090  0.000000  GB          
+1748-202153865.3759513749340 14.72  1949.6090  0.000000  GB          
+1748-202153895.1128342639150 25.14  1949.6090  0.000000  GB          
+1748-202153895.3234693978539 14.65  1949.6090  0.000000  GB          
+1748-202153920.0527417245341 19.29  1949.6090  0.000000  GB          
+1748-202153920.2397147375688 13.25  1949.6090  0.000000  GB          
+1748-202153954.9721608230925 20.71  1949.6090  0.000000  GB          
+1748-202153955.1745617578295 23.57  1949.6090  0.000000  GB          
+1748-202153980.9030418049465 23.45  1949.6090  0.000000  GB          
+1748-202153981.1198134300380 22.16  1949.6090  0.000000  GB          
+1748-202154010.8214331092625 23.53  1949.6090  0.000000  GB          
+1748-202154011.0317678723316 21.01  1949.6090  0.000000  GB          
+1748-202154050.7047431559353 21.66  1949.6090  0.000000  GB          
+1748-202154050.9462470846578 75.30  1949.6090  0.000000  GB          
+1748-202154093.6566052284065 19.65  1949.6090  0.000000  GB          
+1748-202154095.6533073684218 16.28  1949.6090  0.000000  GB          
+1748-202154098.6648705992371 21.93  1949.6090  0.000000  GB          
+1748-202154099.7097847910078 14.00  1949.6090  0.000000  GB          
+1748-202154148.6865194328392 19.35  1949.6090  0.000000  GB          
+1748-202154150.4251333752446 32.92  1949.6090  0.000000  GB          
+1748-202154151.5268221946436 33.83  1549.6090  0.000000  GB          
+1748-202154152.7174473185851118.43  1549.6090  0.000000  GB          
+1748-202154153.5485841298987 16.45  1949.6090  0.000000  GB          
+1748-202154160.5228633859925 30.18  1949.6090  0.000000  GB          
+1748-202154187.3315834930736 21.80  1949.6090  0.000000  GB          
+1748-202154187.5873241699201 20.75  1949.6090  0.000000  GB          
+1748-202154099.7097857411096 32.75  1687.1090  0.000000  GB          
+1748-202154099.7097854257332 31.29  1762.1090  0.000000  GB          
+1748-202154099.7097851481118 37.13  1837.1090  0.000000  GB          
+1748-202154099.7097849032831 37.40  1912.1090  0.000000  GB          
+1748-202154099.7097846849898 35.24  1987.1090  0.000000  GB          
+1748-202154099.7097844915439 50.83  2062.1090  0.000000  GB          
+1748-202154099.7097843152038 38.43  2137.1090  0.000000  GB          
+1748-202154099.7097841593674 48.59  2212.1090  0.000000  GB          
```

Comment:
What is this file for?

Reply:
This is for testing reading the ITOA format. It is used in `test_read_itoa` in `test_toa_reader.py`.
It raises an exception now, but that may change in the future.